terraform/region/data_sources.tf

```diff
@@ -0,0 +1 @@
+data "aws_caller_identity" "current" {}
```

Comment:
What's the purpose of this please?

Reply:
this provides information to terraform for it's current aws context, which can be referenced by the configuration. See: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity